GoogleForCreators/web-stories-wp
# Embeds: fix broken carousel arrow button

## Context



The arrow icon for the stories carousel embed was not working anymore because the CSS referenced an SVG in a wrong path/URL.

## Summary



Fixes the broken carousel arrow button by moving the icon to an `inline-icons` folder so it's referenced as a base64 inlined resource in the CSS and not an external file.

## Relevant Technical Choices



## To-do



## User-facing changes



<img width="455" alt="Screenshot 2022-04-11 at 23 03 30" src="https://user-images.githubusercontent.com/841956/162832099-f90ebca7-3545-4d8e-8ced-a8c66e3c6239.png">

## Testing Instructions






This PR can be tested by following these steps:

1. Add stories carousel on the frontend, for example via the Gutenberg block
2. See arrow icons


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #11224
